test.toml

```diff
@@ -11,6 +11,15 @@ database_url = 'sqlite:///ursabot.sqlite'
   [projects.arrow]
   enabled = true
 
+[reporters]
```

Comment:
@fsaintjacques shouldn't we disable these?

Reply:
done